client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/operations/OperationsFactory.java

```diff
@@ -212,7 +216,8 @@ public SizeOperation newSizeOperation() {
    }
 
    public <T> ExecuteOperation<T> newExecuteOperation(String taskName, Map<String, byte[]> marshalledParams) {
-      return new ExecuteOperation<>(codec, transportFactory, cacheNameBytes, topologyId, flags(), clientIntelligence, taskName, marshalledParams);
+      return new ExecuteOperation<>(codec, transportFactory, cacheNameBytes == null ? DEFAULT_CACHE_NAME_BYTES : cacheNameBytes,
+            topologyId, flags(), clientIntelligence, taskName, marshalledParams);
```

Comment:
Can we not assign `cacheNameBytes` to `DEFAULT_CACHE_NAME_BYTES` in the constructor if it was null instead and just always have this variable as non null?